src/TicketingSystem.PaymentsApi/settings.Development.json

```diff
@@ -0,0 +1,6 @@
+{
+  "ConnectionStrings": {
+    "connectionString": "mongodb+srv://t1ckets4dm:va2ehglAG%40%23EECV%26@ticketing-system.mongocluster.cosmos.azure.com/?tls=true&authMechanism=SCRAM-SHA-256&retrywrites=false&maxIdleTimeMS=120000"
```

Comment:
Is this intended to use different DB for payments?

Reply:
It was not supposed to be here at all 😮
Updated gitignore in a proper way